src/sentry/integrations/github/client.py

```diff
@@ -464,12 +464,12 @@ def get_user(self, gh_username: str) -> Any:
         """
         return self.get(f"/users/{gh_username}")
 
-    def get_labels(self, repo: str) -> Sequence[Any]:
+    def get_labels(self, owner: str, repo: str) -> list[Any]:
         """
-        Fetches up to the first 100 labels for a repository.
+        Fetches all labels for a repository.
         https://docs.github.com/en/rest/issues/labels#list-labels-for-a-repository
         """
-        return self.get(f"/repos/{repo}/labels", params={"per_page": 100})
```

Comment:
It was very confusing to just see `repo` being replaced when the API docs show `owner` and `repo`.